config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
Is this config file needed? I'm unsure what it is. but it sounds web sockets related, which I'm not aware of smart-answers using.